docs/modules/deployment/pages/core/admin-password-reset.adoc

```diff
@@ -6,7 +6,7 @@
 This section explains how to set the web UI admin password using the {page-component-title} installation script.
 
 After you install {page-component-title}, the web UI admin user's password is set to `admin` by default.
-You can change this password after the initial login to the {page-component-title} web UI.
+You can change this password the first-time you sign in to the {page-component-title} web UI, and subsequently using the *Change Password* option in the UI.
```

Comment:
May need to tweak my wording to indicate the option is in the user menu of the menu bar.
```suggestion
You will be prompted to change this password the first time you sign in to the {page-component-title} web UI.
You can also change the password by using the *Change Password* option in the UI any time you are logged in.
```